orbit/pkg/update/update.go

```diff
@@ -70,6 +70,15 @@ func (ts Targets) SetTargetChannel(target, channel string) {
 	ts[target] = t
 }
 
+// SetExtensionsTargetInfo sets/updates the TargetInfo for extension targets
+func (u *Updater) SetExtensionsTargetInfo(name, platform, channel, file string) {
```

Comment:
Same as `UpdateRunnerOptTargets`:

Looks like there's a race now.

We should now protect `r.opt.Targets` with a lock.

Reply:
updated, now using a mutex